torch/csrc/jit/python_custom_class.cpp

```diff
@@ -0,0 +1,71 @@
+#include <torch/csrc/jit/python_custom_class.h>
+#include <torch/csrc/jit/script/sugared_value.h>
+
+namespace torch {
+namespace jit {
+
+struct CustomMethodProxy;
+struct CustomObjectProxy;
+
+void initPythonCustomClassBindings(PyObject* module) {
+  auto m = py::handle(module).cast<py::module>();
+
+  m.def("_get_custom_class_python_wrapper", [](const std::string& str) {
```

Comment:
this really needs a comment, I spent 5 minutes trying to figure out what it does.

Reply:
Also probably misnamed. It retries the python wrapper for the _constructor_ of the class, not the class itself.

Reply:
The analogue to Python is the code object, so that's why I called it class wrapper and not constructor wrapper. I can still change it if you want